libplen/src/constants.rs

```diff
@@ -9,6 +9,7 @@ pub const POWERUP_BOUNCE_HEIGHT: f32 = 10.;
 
 // currently hardcoded to the background image size
 pub const WORLD_SIZE: f32 = 3000.;
+pub const DELTA_TIME: f32 = 0.01;
```

Comment:
Sleep duration [here](https://github.com/HannesTuhkala/plen/blob/master/src/server.rs#L91) should probably also use this

Reply:
Fixed in next commit